cli/foundation-models/system/inference/text-to-image/batch-deploy.yml

```
@@ -1,5 +1,5 @@
 $schema: https://azuremlschemas.azureedge.net/latest/batchDeployment.schema.json
-description: "Batch endpoint for for text-to-image task"
+description: "Batch endpoint for for image-text-to-image task"
```

This is a common file. 

```suggestion
description: "Batch endpoint for for (image-)text-to-image task"
```